Formula/sonar-scanner.rb

```diff
@@ -0,0 +1,32 @@
+class SonarScanner < Formula
+  desc "Launcher to analyze a project with SonarQube"
+  homepage "http://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner"
+  url "https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-2.8.zip"
+  version "2.8"
+  sha256 "0295365a7e5d4499ec6b46cb6c70f3fa127159b58b73930f675acd0897a6b350"
+  head "https://github.com/SonarSource/sonar-scanner-cli.git"
+
+  bottle :unneeded
+
+  def install
+    # Remove windows files
+    rm_rf Dir["bin/*.bat"]
+    libexec.install Dir["*"]
+    bin.install libexec/"bin/sonar-scanner"
+
+    # sonar-sanner uses this ENV to work out where to install.
+    ENV["SONAR_SCANNER_HOME"] = libexec
```

Comment:
The env wrapper script should take care of that, but it's not being run in the `def install` block, so it shouldn't be necessary there.